src/main/java/uk/gov/hmcts/reform/professionalapi/controller/response/OrganisationByProfileResponse.java

```diff
@@ -0,0 +1,25 @@
+package uk.gov.hmcts.reform.professionalapi.controller.response;
+
+import lombok.Getter;
+import lombok.Setter;
+import uk.gov.hmcts.reform.professionalapi.domain.Organisation;
+import uk.gov.hmcts.reform.professionalapi.util.RefDataUtil;
+
+import java.time.LocalDateTime;
+import java.util.List;
+
+@Setter
+@Getter
+public class OrganisationByProfileResponse {
+    private String organisationIdentifier;
+    private String status;
+    private LocalDateTime lastUpdated;
+    private List<String> organisationProfileIds;
+
+    public OrganisationByProfileResponse(Organisation organisation) {
+        this.organisationIdentifier = organisation.getOrganisationIdentifier();
```

Comment:
what is the possibility of organisation is null here?

Reply:
This was a result due to something that happened in an integration test